contracts/zksync/test/claim_payment.sh

```diff
@@ -4,49 +4,32 @@
 
 printf "${GREEN}\n=> [ETH] Making Claim Payment${COLOR_RESET}\n"
```

Comment:
This file must follow the same sintax than `.github/workflows/scripts/claim_payment.sh`